src/azure-cli/azure/cli/command_modules/appconfig/_azconfig/models.py

```diff
@@ -43,6 +43,14 @@ def __init__(self, key, value=None, label=None, tags=None, content_type=None):
         self.last_modified = None
         self.locked = None
 
+    def __str__(self):
+        return "\nKey: " + self.key + \
+               "\nValue: " + self.value + \
+               "\nLabel: " + (self.label if self.label else '') + \
+               "\netag: " + self.etag + \
+               "\nLast Modified: " + self.last_modified + \
```

Comment:
Do we want to show etag? Also we should add content-type as well.

Reply:
I removed etag, added content type to the JSON output. This is now just being printed to the log file, in case something fails and not shown to the user. I'm adding content type here, and keeping etag, since it might help diagnose things in case of failure.